pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -199,6 +199,9 @@ public static class ControllerPeriodicTasksConf {
     // RealtimeSegmentRelocator has been rebranded as SegmentRelocator
     public static final String DEPRECATED_REALTIME_SEGMENT_RELOCATION_INITIAL_DELAY_IN_SECONDS =
         "controller.realtimeSegmentRelocation.initialDelayInSeconds";
+    public static final String REBALANCE_CHECKER_INITIAL_DELAY_IN_SECONDS =
+        "controller.rebalanceChecker.initialDelayInSeconds";
+    public static final String REBALANCE_CHECKER_CHECK_ONLY = "controller.rebalanceChecker.checkOnly";
```

Comment:
This config will void the retry from the rebalance call, so I definitely won't make it the default. IMO we can just remove it, and if by any change rebalance checker has a bug, we can turn it completely off

Reply:
I'm fine with removing this feature flag to be simple. I thought there might be cases where users just want to alert when this checker task detects rebalance failure, but retry the rebalance manually.